libs/blocks/marketo/marketo.js

```diff
@@ -123,6 +127,16 @@ export const formSuccess = (formEl, formData) => {
   return false;
 };
 
+export function setProductOfInterest(formData, search = window.location.search) {
+  if (formData[PROGRAM_POI]) return;
+
+  const queryPoi = new URLSearchParams(search).get(QUERY_POI);
+  const sessionPoi = sessionStorage.getItem(SESSION_POI);
+  const productOfInterest = queryPoi || sessionPoi || null;
+
+  formData[PROGRAM_POI] = productOfInterest;
```

Comment:
Is it alright to set null in this case?

Reply:
Thanks, `productOfInterest` will already be null, I should only set `formData` if `productOfInterest` is truthy.